src/app/game/interface/items-interface/items-interface.component.ts

```diff
@@ -16,15 +16,23 @@ export class ItemsInterfaceComponent implements OnInit {
   scrollable = true;
 
   ngOnInit(): void {
-    this.items.onUseItem = this.onItemUse.bind(this);
-    this.updateItemData();
+    console.warn('ngOnInit Items');
```

Comment:
Da es sich hierbei um eine Init funktion handelt, würde ich dir abraten `console.warn()` zu benutzen. In dem Falle bietet sich `console.log()` besser an.

Reply:
auch ein überbleibsel